architecture/configuration.md

```diff
@@ -33,6 +33,9 @@ export default {
   experimental: {
     typescript: true,
   },
+  dependencies: {
```

Comment:
are we going to be manually managing all vendor dependencies?

Reply:
@jssee no, we a strategy alternative to `package.json` as we will allow a `spike.exe` workflow w/o a node.js project.

So the idea is that you can include arbitrary source code on top of node_modules to be converted to `web_modules`, in addition to `package.json`.